acceptance/tests/language/class_inheritance.rb

```diff
@@ -0,0 +1,19 @@
+test_name 'C14943: Class inheritance works correctly' do
+
+  agents.each do |agent|
+    test_manifest = <<MANIFEST
+      class bar { notice("This is class $name") }
+      class foo::bar { notice("This is class $name") }
+      class foo inherits bar { notice("This is class $name") }
+
```

Comment:
You are relying on $name being correctly set - here I think it would be better with hardcoded names since the purpose is not to test that $name is set to the classname. I am being pedantic, feel free to ignore.